.github/workflows/continous-integration.yml

```diff
@@ -165,6 +165,10 @@ jobs:
       run: |
         make actions &
 
+    - name: Run duckling server
+      run: |
+        docker run --name duckling_container -d -p 8000:8000 rasa/duckling
```

Comment:
If this container does not need to be preserved after it is stopped command should include `--rm` flag, which will remove the container once it is stopped. Helps to keep things clean.

Reply:
Nice, will do.

Reply:
Addressed: https://github.com/RasaHQ/rasa-calm-demo/pull/30/commits/b6ccb2d7ad49db3f081e6776beddc6bbfb5e4526